protocol/avdl/prove_common.avdl

```diff
@@ -1,5 +1,5 @@
 @namespace("keybase.1")
-protocol prove_common {
+protocol proveCommon {
```

Comment:
this is the only one that isn't camelcase, @maxtaco said i should make it conform